packages/docusaurus-theme-classic/src/theme/Layout/index.tsx

```diff
@@ -42,6 +42,7 @@ export default function Layout(props: Props): JSX.Element {
       <Navbar />
 
       <div
+        id="main-wrapper"
```

Comment:
In any case I'd rather have this ID added as a constant in `@docusaurus/theme-common`

Make make its purpose more specific with a name like "skipToContentId" ?

I'd also make use of it in the JS selector instead of the existing className

Reply:
Comments addressed. Thanks for the review!